haas/rest.py

```diff
@@ -164,10 +172,19 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        if isinstance(methods, list):
```

Comment:
I assume not changing every ``method`` into ``list`` can help us keep the rest of codebase untouched.